assets/js/block-extensions/core-query.js

```diff
@@ -0,0 +1,273 @@
+/* eslint-disable @wordpress/no-unsafe-wp-apis */
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/extensions */
+/**
+ * External dependencies
+ */
+import { v4 as uuidv4 } from 'uuid';
+import { registerBlockExtension, ContentPicker } from '@10up/block-components';
```

Comment:
If we update to version 1.21.1 of the block components we can use the new code split import syntax for these also: 

```suggestion
import { ContentPicker } from '@10up/block-components/components/content-picker';
import { registerBlockExtension } from '@10up/block-components/api/register-block-extension';
```

which will result in a smaller bundle size :)